lib/src/cartesian/components/CartesianAxis.tsx

```diff
@@ -112,7 +119,9 @@ export const CartesianAxis = <
   const [x1r = 0, x2r = 0] = xScale.range();
   const fontSize = font?.getSize() ?? 0;
 
-  const yAxisNodes = yScale.ticks(yTicks).map((tick) => {
+  const yAxisNodes = (
+    yTickValues ? downsampleTicks(yTickValues, yTicks) : yScale.ticks(yTicks)
```

Comment:
this is just mine own personal nit, but I'd suggest we break this out into a variable just for readability like:

`const yTicksSomethingName = yTickValues ? downsampleTicks(yTickValues, yTicks) : yScale.ticks(yTicks)`

And maybe a comment?